JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1055,6 +1076,65 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(p)
 }
 
+
+.descriptivesDotPlots <- function(dataset, options, variable){
+  
+  
+  if (options$splitby != "" ) {
+    # return a collection
+    split <- names(dataset)
+    
+    plotResult <- createJaspContainer(title = variable)
+    plotResult$dependOn(optionContainsValue = list(variables = variable))
+    
+    for (l in split)
+      plotResult[[l]] <- .descriptivesDotPlots_SubFunc(dataset = dataset[[l]], variable = variable, title = l)
+    
+    return(plotResult)
+    
+  }else{
+    
+    dotplot <- .descriptivesDotPlots_SubFunc(dataset = dataset, variable = variable, title = variable)
+    dotplot$dependOn(options="splitby", optionContainsValue = list(variables = variable))
```

Comment:
```suggestion
    dotplot$dependOn(optionContainsValue = list(variables = variable))
```
Since we have `jaspResults[["DotPlots"]]$dependOn(c("splitby", "descriptivesDotPlot"))`, the dotplot already depends on `"splitby"`, so this dependency is superfluous.